[2024-10-16T09:40:47.823+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-10-13T00:00:00+00:00 [queued]>
[2024-10-16T09:40:47.834+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-10-13T00:00:00+00:00 [queued]>
[2024-10-16T09:40:47.835+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:47.836+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:40:47.836+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:40:47.855+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-10-13 00:00:00+00:00
[2024-10-16T09:40:47.864+0000] {standard_task_runner.py:55} INFO - Started process 4517 to run task
[2024-10-16T09:40:47.870+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-10-13T00:00:00+00:00', '--job-id', '476', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmp9ztxhbql']
[2024-10-16T09:40:47.871+0000] {standard_task_runner.py:83} INFO - Job 476: Subtask insert_rate
[2024-10-16T09:40:47.981+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-10-13T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:40:48.020+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:40:48.079+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-10-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-10-13T00:00:00+00:00
[2024-10-16T09:40:48.080+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-10-13')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:40:48.092+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:48.304+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:40:48.318+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUS', None, '2024-10-13')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:40:48.320+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 338, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 369, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedColumn: column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-10-13')
                              ^

[2024-10-16T09:40:48.330+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20241013T000000, start_date=20241016T094047, end_date=20241016T094048
[2024-10-16T09:40:48.343+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 476 for task insert_rate (column "none" does not exist
LINE 3: VALUES ('USD', 'RUS', None, '2024-10-13')
                              ^
; 4517)
[2024-10-16T09:40:48.382+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-10-16T09:40:48.402+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T09:50:34.194+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-10-13T00:00:00+00:00 [queued]>
[2024-10-16T09:50:34.208+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-10-13T00:00:00+00:00 [queued]>
[2024-10-16T09:50:34.209+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:50:34.209+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:50:34.210+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:50:34.226+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-10-13 00:00:00+00:00
[2024-10-16T09:50:34.231+0000] {standard_task_runner.py:55} INFO - Started process 7349 to run task
[2024-10-16T09:50:34.236+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_rus_dag', 'insert_rate', 'scheduled__2024-10-13T00:00:00+00:00', '--job-id', '1341', '--raw', '--subdir', 'DAGS_FOLDER/RUS_DAG.py', '--cfg-path', '/tmp/tmpuofc47nn']
[2024-10-16T09:50:34.237+0000] {standard_task_runner.py:83} INFO - Job 1341: Subtask insert_rate
[2024-10-16T09:50:34.321+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_rus_dag.insert_rate scheduled__2024-10-13T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:50:34.368+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:50:34.432+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_rus_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-10-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-10-13T00:00:00+00:00
[2024-10-16T09:50:34.433+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 96.24457946, '2024-10-13')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:50:34.444+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:50:34.694+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:50:34.702+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'RUB', 96.24457946, '2024-10-13')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:50:34.704+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:50:34.717+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_rus_dag, task_id=insert_rate, execution_date=20241013T000000, start_date=20241016T095034, end_date=20241016T095034
[2024-10-16T09:50:34.743+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:50:34.764+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
